tests/models/test_mistral.py

```diff
@@ -3,14 +3,15 @@
 Run `pytest tests/models/test_mistral.py --forked`.
 """
 import pytest
+from vllm.model_executor.parallel_utils.parallel_state import destroy_model_parallel
 
 MODELS = [
     "mistralai/Mistral-7B-Instruct-v0.1",
 ]
 
 
 @pytest.mark.parametrize("model", MODELS)
-@pytest.mark.parametrize("dtype", ["bfloat16"])
+@pytest.mark.parametrize("dtype", ["half"])
```

Comment:
Mistral model uses bfloat16 as specified [here](https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.1/blob/9ab9e76e2b09f9f29ea2d56aa5bd139e4445c59e/config.json#L20).

Reply:
Yes. Thanks.

I don't know why I had to change it to `half` this afternoon. But after git pull upstream, it seem bffloat16 works now.